packages/lesswrong/lib/modules/alignment-forum/posts/custom_fields.js

```diff
@@ -54,4 +55,30 @@ Posts.addField([
     }
   },
 
+  {
+    fieldName: 'afSticky',
+    fieldSchema: {
+      order: 10,
+      type: Boolean,
+      optional: true,
+      label: "Alignment Sticky",
+      defaultValue: false,
+      group: formGroups.admin,
+      viewableBy: ['guests'],
+      editableBy: ['alignmentForumAdmins', 'admins'],
+      insertableBy: ['alignmentForumAdmins', 'admins'],
+      control: 'checkbox',
+      onInsert: (post) => {
+        if(!post.afSticky) {
+          return false;
+        }
+      },
+      onEdit: (modifier, post) => {
+        if (!modifier.$set.afSticky) {
```

Comment:
Maybe it's possible to have a modifier that only has an `$unset` field defined, but no `$set` field? Then this would crash.